examples/text_to_image/README.md

```diff
@@ -52,7 +52,7 @@ If you have already cloned the repo, then you won't need to go through these ste
 With `gradient_checkpointing` and `mixed_precision` it should be possible to fine tune the model on a single 24GB GPU. For higher `batch_size` and faster training it's better to use GPUs with >30GB memory.
 
 **___Note: Change the `resolution` to 768 if you are using the [stable-diffusion-2](https://huggingface.co/stabilityai/stable-diffusion-2) 768x768 model.___**
-
+<!-- accelerate_snippet_start -->
```

Comment:
use comment as a marker. In markdowns, you can use html style commenting `<!-- xyz -->`